src/compass/s1_rdr2geo.py

```diff
@@ -116,7 +116,12 @@ def run(cfg, burst=None, save_in_scratch=False):
                        rdr2geo_cfg.compute_local_incidence_angle,
                        gdal.GDT_Float32),
                        'heading': (
-                       rdr2geo_cfg.compute_azimuth_angle, gdal.GDT_Float32)}
+                       rdr2geo_cfg.compute_azimuth_angle, gdal.GDT_Float32),
```

Comment:
Not related to this changes here - Can you change the comment above at 106 to something like [this](https://github.com/opera-adt/COMPASS/blob/e3518a9ac22b6461870dd88ecb917bc250c958ae/src/compass/s1_geocode_metadata.py#L130-L132). My not as nicely formatted commented below:
```
save output rasters to dict with key as topo layer file name and item as dict of bool to enable/disable and datatype
```
My original comment is pretty close to useless.